src/index.js

```diff
@@ -1,6 +1,61 @@
 import './style.css'
 
-console.log('It works!')
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+// React.createElement
+function CreateElementEx() {
+  return React.createElement(
+    'div',
+    { className: 'component' },
+    'Hello from ',
+    React.createElement('strong', {}, 'React.createElement')
+  )
+}
+
+// React.Component
+class ComponentEx extends React.Component {
+  render() {
+    return (
+      <div className="component">
+        Hello from <strong>React.Component</strong>
+      </div>
+    )
+  }
+}
+
+// React.PureComponent
+class PureComponentEx extends React.Component {
```

Comment:
Pure?

Reply:
Sorry, I should've used `React.PureComponent`